posts/2021-02-15-eclipse-transformer.adoc

```diff
@@ -0,0 +1,322 @@
+---
+layout: post
+title: "Using the Eclipse Transformer for Jakarta EE 9 Development and Test"
+categories: blog
+author_picture: TBD
+author_github: https://github.com/tbitonti
+seo-title: Using the Eclipse Transformer for Jakarta EE 9 Development and Test
+seo-description: The Eclipse Transformer, an Eclipse hosted open source project, provides a tool which can be used to quickly create Jakarta EE 9 enabled artifacts from existing Jakarta EE 8 artifacts.  Artifacts which can be transformed include applications in source and binary form, test code, and Open Liberty server configurations.  This article provides a brief introduction to the Eclipse Transformer and shows how to use the Eclipse Transformer command line interface.
+blog_description: "The Eclipse Transformer, an Eclipse hosted open source project, provides a tool which can be used to quickly create Jakarta EE 9 enabled artifacts from existing Jakarta EE 8 artifacts.  Artifacts which can be transformed include applications in source and binary form, test code, and Open Liberty server configurations.  This article provides a brief introduction to the Eclipse Transformer and shows how to use the Eclipse Transformer command line interface."
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Using the Eclipse Transformer for Jakarta EE 9 Development and Test
+Thomas Bitonti <https://github.com/tbitonti>
+
+== Overview
+
+This article describes how application developers can use the Eclipse Transformer to update their application and test artifacts to run using Jakarta EE 9.
+
+The core problem introduced by Jakarta EE 9 is the renaming of package prefixes from "javax" to "jakarta" for APIs and properties.  As a consequence, existing applications which run against Java EE 7 or which run against Jakarta EE 8 do not run against Jakarta EE 9.  To run existing applications as Jakarta EE 9 applications, references to specific packages must be updated.
+
+To solve this problem, the Open Liberty development team created the Eclipse Transformer.  This open source project provides a utility which updates package references in a variety of artifacts.
+
+The development team uses the Eclipse Transformer to update bundle implementation archives.  The team also uses the transformer to update test artifacts, which include:
+
+* Test applications;
```

Comment:
I don't think you need the semicolons here.

Reply:
Updated.